### PR TITLE
[COST-5755] Enabling Slack notifications on build pipelines - pushes to main

### PR DESCRIPTION
**Pull request by bacciotti**

### CODE REVIEW

File: .tekton/push.yaml [hunk 1]
```diff
@@ -41,3 +41,22 @@ spec:
...
+          value: |
+            :x: Pipeline `<$(context.pipelineRun.name)>` has failed or completed with issues.
+            See details: <https://console.redhat.com/application-pipeline/workspaces/cost-mgmt-dev/applications/nise-populator/pipelineruns/$(context.pipelineRun.name)|Pipeline Run>.
+        - name: secret-name
+          value: "slack-webhook-notification-secret"
+        - name: key-name
+          value: "webhook-url"
+      taskRef:
+        name: slack-webhook-notification
+        bundle: quay.io/konflux-ci/tekton-catalog/task-slack-webhook-notification:0.1@sha256:dc17b70633363d78414b8c06dc1660d25742935f106a6116995638e1210c2730
```
Comment (samdoran) [inline .tekton/push.yaml]: I think this taskRef might be incorrect
```suggestion
        resolver: bundles
        params:
          - name: name
              value: slack-webhook-notification
          - name: bundle
            value: quay.io/konflux-ci/tekton-catalog/task-slack-webhook-notification:0.1@sha256:dc17b70633363d78414b8c06dc1660d25742935f106a6116995638e1210c2730
            - name: kind
               value: task
```

File: .tekton/push.yaml [hunk 2]
```diff
@@ -41,3 +41,28 @@ spec:
     - name: git-auth
       secret:
         secretName: '{{ git_auth_secret }}'
+
+  finally:
```
Comment (samdoran) [inline .tekton/push.yaml:45]: This task needs to go in the `finally` section of the Pipeline, not the PipelineRun.

**Comment by bacciotti** [comment]
Closing this one because EC run is failing continuously. The same change was made on this new PR: https://github.com/project-koku/nise-populator/pull/183.

**Pull request closed without merging**